docs/docs-content/vm-management/create-manage-vm/advanced-topics/advanced-topics.md

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_label: "Advanced Topics"
```

Comment:
I wonder if this should be titled `VMO Customizations` since I agree that it provides customizations